commands/create.go

```diff
@@ -513,31 +539,72 @@ func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, machineName, userdataFla
 			return err
 		}
 	}
+	log.Infof("[updateUserdataFile] userdataFile has contents [%s]", userdataFile)
 
 	customScriptContent, err := ioutil.ReadFile(customInstallScript)
 	if err != nil {
 		return err
 	}
 	// Remove the shebang
 	customScriptContent = regexp.MustCompile(`^#!.*\n`).ReplaceAll(customScriptContent, nil)
+	log.Infof("[updateUserdataFile] customScriptContent has contents [%v]", customScriptContent)
```

Comment:
Same. There may be sensitive information in this and logging it is not a good idea.